scripts/releases/prebuild/swift-package.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall react_native
+ */
+
+const {RESOURCES_FOLDER, TARGET_FOLDER} = require('./constants');
+const fs = require('fs');
+const path = require('path');
+
+/*::
+  import type {Dependency} from './types';
+  import {skip} from "rxjs/operators";
```

Comment:
why do we need this?

Reply:
We don't I'll remove it. I think it was an auto-import that failed.